src/components/Buttons/ButtonBase/index.tsx

```diff
@@ -2,8 +2,12 @@ import { Button as MaterialButton, ButtonProps as MaterialButtonProps } from "@m
 
 export interface ButtonBaseProps extends MaterialButtonProps {}
 
+const buttonStyle = {
+    borderRadius: "calc(1em + 6px)",

```

Comment:
the button uses the font of typography button and it uses rem so we should probably use rem instead of em